src/core/show.js

```diff
@@ -48,21 +48,39 @@ const _getBody = options => {
     return $html;
 };
 
+const __appendElementToBody = (onTop, $element) => {
+    if(onTop) {
```

Comment:
Missing space after keyword

Reply:
Why double lodash before function name?

Reply:
This is the code style of this module. Double lodash shows that is a private not exported function